drivers/adc/adc_ite_it8xxx2.c

```diff
@@ -30,7 +30,7 @@ LOG_MODULE_REGISTER(adc_ite_it8xxx2);
 /* ADC channels disabled */
 #define IT8XXX2_ADC_CHANNEL_DISABLED 0x1F
 /* ADC sample time delay (Unit:us) */
-#define IT8XXX2_ADC_SAMPLE_TIME_US 200
+#define IT8XXX2_ADC_SAMPLE_TIME_US 500
```

Comment:
Why is this change required? Is this sample time delay also valid on the `it8xxx2` board?

Reply:
There is an explanation in line 436 for the change in sampling time, which also applies to it8xxx2_evb board.

Reply:
Got it, thanks